Jenkinsfile

```diff
@@ -37,13 +37,13 @@ pipeline {
 		stage('build') {
 			steps {
 				nvm(getNodejsVersion()) {
-					withEnv(["DEFAULT_NETWORK=testnet"]) {
+					withEnv(["DEFAULT_NETWORK=customNode"]) {
```

Comment:
we should not do this because any user who uses the route from the deployment the default network they would access would be `customNetwork` which would not be available for them.